demos/mqtt/mqtt_demo_basic_tls/demo_config.h

```diff
@@ -43,6 +43,38 @@
 
 /************ End of logging configuration ****************/
 
+/**
+ * @brief MQTT server host name.
+ *
+ * This demo uses the Mosquitto test server. This is a public MQTT server; do not
+ * publish anything sensitive to this server.
```

Comment:
We should add a comment here stating that mosquitto server can downloaded and used in local setup from <webstite> , in case public server in inaccessible or not desirable.

Reply:
Sounds good. Will add that. Thanks.